lib/types.d.ts

```diff
@@ -11,11 +11,16 @@ declare global {
   type JSXAttribute = ASTNode;
   type JSXElement = ASTNode;
   type JSXFragment = ASTNode;
+  type JSXOpeningElement = ASTNode;
   type JSXSpreadAttribute = ASTNode;
 
-  type Context = eslint.Rule.RuleContext
+  type Context = eslint.Rule.RuleContext;
 
-  type TypeDeclarationBuilder = (annotation: ASTNode, parentName: string, seen: Set<typeof annotation>) => object;
+  type TypeDeclarationBuilder = (
```

Comment:
This was auto-formatted, I assume this is ok?

Reply:
nbd, i reverted it. i'd suggest disabling autoformatting in your editor :-)